test/functional/query_test.coffee

```diff
@@ -101,6 +101,13 @@ describe 'Vertica.Connection#query', ->
       assert.ok typeof err, 'string'
       done()
 
+    it "should return multi-queries error", (done) ->
+      sql = "SELECT 1; SELECT 1"
+      connection.query sql, (err, _) ->
+        assert.ok typeof err, 'string'
+        assert.equal err.message, "Cannot handle multi-queries with a callback!"
+      done()
```

Comment:
Also, I think the connection will be in a broken state after this. Is that acceptable?

 ``` coffeescript
connection.query "SELECT 1; SELECT 1", (err, _) ->
  assert.ok typeof err, 'string'
  assert.equal err.message, "Cannot handle multi-queries with a callback!"
  connection.query "SELECT 1", (err, result) ->
    // should this work?
```